src/foremast/pipeline/create_pipeline.py

```diff
@@ -127,6 +127,7 @@ def render_wrapper(self, region='us-east-1'):
                 'slack': slack,
                 'root_volume_size': root_volume_size,
                 'ami_template_file': ami_template_file,
+                'pipeline': self.settings['pipeline']
```

Comment:
Would i make sense to add this under its own `pipeline` key, instead as part of `app`?
https://github.com/gogoair/foremast/pull/266/files#diff-f6fa73849891e3ceadc9aed00baa386aR119

Reply:
Yeah, that would make sense, except I was just trying to be consistent with the rest of the code. For the pipeline stage blocks we also nest this under app.pipeline

Reply:
Would that cause conflict/confusion though?
In 1 case, you have app.pipeline that has environment specific configs (`application-master-$env.json`).
In the other case app.pipeline could be the app leverl configs (`pipeline.json`).

Reply:
I'm not saying the approach is wrong, I'm just thinking of any downstream template ramifications in doing this.

Reply:
To add, I really like having the contents available in `pipeline.json` to all the templates.

Reply:
yeah, this is poor implementation from that past.

`app.pipeline` in both cases contains the same thing, the contents of `pipeline.json`. We render this wrapper template with different variables for some reason.